src/analysis/specific_limits.lean

```diff
@@ -231,13 +231,12 @@ begin
   apply nnreal.tendsto_pow_at_top_nhds_0_of_lt_1 hr
 end
 
-lemma tendsto_pow_at_top_nhds_0_of_norm_lt_1 {K : Type*} [normed_field K] {ξ : K}
-  (_ : ∥ξ∥ < 1) : tendsto (λ n : ℕ, ξ^n) at_top (𝓝 0) :=
+/-- In a normed ring, the powers of an element x with `∥x∥ < 1` tend to zero. -/
+lemma tendsto_pow_at_top_nhds_0_of_norm_lt_1 {α : Type*} [normed_ring α] {x : α}
```

Comment:
Can you use `R` for a ring? I know this is not always the case in this file, but getting it right for the new lemmas can only help!

Reply:
Can you clarify, should I make this change for my lemmas about rings in this file only, or in every file?

Reply:
Just for your lemmas, and wherever you see fit.

Reply:
OK, I made the change to my own lemmas in this file (`analysis/specific_limits`), and throughout `analysis/normed_space/units`.